packages/trees/frontend/sirius-components-trees/src/views/ExplorerView.tsx

```diff
@@ -64,15 +66,15 @@ export const ExplorerView = ({ editingContextId, selection, setSelection, readOn
 
   const [{ value, context }, dispatch] = useMachine<ExplorerViewContext, ExplorerViewEvent>(explorerViewMachine);
   const { toast, explorerView } = value as SchemaValue;
-  const { id, tree, expanded, maxDepth, synchronized, message } = context;
+  const { id, tree, expanded, maxDepth, synchronized, isSynchronizedToggled, message } = context;
```

Comment:
Having both `synchronized` and `isSynchronizedToggled` is confusing. They should probably be given more meaninful names.